docs/application/tizen-studio/native-tools/ui-builder/component-attributes.md

```diff
@@ -1236,11 +1236,11 @@ Some properties are supported only in specific profiles. The following table lis
 | Toolbar  item           | src                        | Yes               | Yes        | Yes            | No                     | No                           | No                       |
 | Toolbar item            | selected                   | Yes               | Yes        | Yes            | No                     | No                           | No                       |
 
-## Component Attributes – Custom UI Component
+## Component Attributes - Custom UI Component
 
-The `&lt;customcomponent&gt;` is a customized element.
+The `<customcomponent>` is a customized element.
 
-### Custom UI Component (![Custom UI Component](file:///D:/git/online-doc/org.tizen.studio/html/new/images/component_attributes_custom_icon.png))
+### Custom UI Component (![Custom UI Component](media/component_attributes_custom_icon.png))
 
 A [custom UI component](ui-builder-customcomponent.md) is a component you can create through EDC.
```

Comment:
It looks `custom-ui-component.md` is correct.

Reply:
done